diff --git a/launcher/ui/pages/global/AccountListPage.cpp b/launcher/ui/pages/global/AccountListPage.cpp
index a4f4dfb9..b0fe13cc 100644
--- a/launcher/ui/pages/global/AccountListPage.cpp
+++ b/launcher/ui/pages/global/AccountListPage.cpp
@@ -188,6 +188,7 @@ void AccountListPage::on_actionAddMicrosoft_triggered()
 
 void AccountListPage::on_actionAddOffline_triggered()
 {
+/*
     if (!m_accounts->anyAccountIsValid()) {
         QMessageBox::warning(
             this,
@@ -200,7 +201,7 @@ void AccountListPage::on_actionAddOffline_triggered()
         );
         return;
     }
-
+*/
     MinecraftAccountPtr account = OfflineLoginDialog::newAccount(
         this,
         tr("Please enter your desired username to add your offline account.")
